src/domain/metadata/usecases/MetadataSyncUseCase.ts

```diff
@@ -41,18 +49,27 @@ export class MetadataSyncUseCase extends GenericSyncUseCase {
             const metadataRepository = await this.getMetadataRepository();
             const syncMetadata = await metadataRepository.getMetadataByIds(ids);
             const elements = syncMetadata[collectionName] || [];
+            const defaultIds = await metadataRepository.getDefaultIds();
 
             for (const element of elements) {
                 //ProgramRules is not included in programs items in the response by the dhis2 API
                 //we request it manually and insert it in the element
                 const fixedElement =
                     type === "programs" ? await this.requestAndIncludeProgramRules(element as Program) : element;
 
+                // In DHIS 2.38.5 and above the defaults parameter is not working
+                // /api/metadata.json?fields=id&filter=id:eq:data_element_id&categoryCombo&defaults=INCLUDE/EXCLUDE; this is not working
+                // so manually removing the default categoryCombo
+                const elementWithoutDefaults =
```

Comment:
A use case should not make any reference to DHIS2 (that's an implementation detail). If DHIS2 has a bug (Indeed, I've checked it: 2.36 OK, 2.38 FAIL), any extra processing should be done in the data layer, probably the metadata DHIS2 repositoriy (or any other file which calls the API with that option `defaults: boolean`  -> I've found also utils/synchronization.ts)

Note: checking dhis2-core, DefaultCategoryService.java, the models that have a default object are the 4 expected: category, categoryOption, categoryCombo, categoryOptionCombo.